tests/search/includes/classes/queue/test-class-cron.php

```diff
@@ -2,20 +2,25 @@
 
 namespace Automattic\VIP\Search\Queue;
 
-use Automattic\VIP\Search\Queue\Cron as Cron;
+use Automattic\VIP\Search\Queue\Cron;
+use Automattic\VIP\Search\Search;
 use PHPUnit\Framework\MockObject\MockObject;
 use WP_UnitTestCase;
 use wpdb;
 
 class Cron_Test extends WP_UnitTestCase {
+	/** @var Search */
+	private $es;
+
 	public function setUp(): void {
 		if ( ! defined( 'VIP_SEARCH_ENABLE_ASYNC_INDEXING' ) ) {
 			define( 'VIP_SEARCH_ENABLE_ASYNC_INDEXING', true );
 		}
 
 		require_once __DIR__ . '/../../../../../search/search.php';
 
-		$this->es = \Automattic\VIP\Search\Search::instance();
+		$this->es = Search::instance();
+		$this->es->init();
```

Comment:
We still need to run `init()`. It is possible that the instance of `Search` has already been created; in this case, `instance()` will just return it. However, WPTL clears all installed hooks and filters, and this is why we have to call `init()` to reinstall them.